integration-tests/src/test/java/oracle/kubernetes/operator/ITMultipleClusters.java

```diff
@@ -93,6 +93,11 @@ public void testCreateDomainTwoConfiguredCluster() throws Exception {
           "createDomainPyScript",
           "integration-tests/src/test/resources/domain-home-on-pv/"
               + TWO_CONFIGURED_CLUSTER_SCRIPT);
+      if ((System.getenv("LB_TYPE") != null && System.getenv("LB_TYPE").equalsIgnoreCase("VOYAGER"))
+          || (domainMap.containsKey("LB_TYPE")
```

Comment:
LB_TYPE is only environment variable. you can look for "loadBalancer" attribute in domain map.